.github/workflows/nodejs.yml

```diff
@@ -57,7 +57,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [14.x, 16.x, 18.x]
+        node-version: [14.x, 16.x, 18.x, 20.x]
```

Comment:
What are the plans re: node 14 which is EOL now ? 
Depending upon those plans, could potentially remove the need for another lib: 
https://github.com/XRPLF/xrpl.js/pull/2273#discussion_r1177267048

Thanks!

Reply:
You'll have to ask some of the other maintainers about that - it's not my responsibility

Reply:
We should drop in 3.0.

Reply:
> You'll have to ask some of the other maintainers 

I see, you just had that air of authority :)